source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.6.0.rst

```diff
@@ -0,0 +1,66 @@
+.. _K8SPXC-1.6.0:
+
+================================================================================
+*Percona Kubernetes Operator for Percona XtraDB Cluster* 1.6.0
+================================================================================
+
+:Date: October 7, 2020
+:Installation: `Installing Percona Kubernetes Operator for Percona XtraDB Cluster <https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html#quickstart-guides>`_
+
+New Features
+================================================================================
+
+* :jirabug:`K8SPXC-416`: :ref:`Support of the proxy-protocol<haproxy-conf-protocol>` in HAProxy
+* :jirabug:`K8SPXC-429`: Possibility to :ref:`restore backups to a new Kubernetes-based environment<backups-restore>`
```

Comment:
This one is wrong. It is about restoring a backup without CR. @delgod

Reply:
Changed to "A possibility to restore backups to a new Kubernetes-based environment with no existing Percona XtraDB Cluster Custom Resource". I think mentioning "new environment" is valuable from the user's point of view (actual use cases of the feature).

Also K8SPXC-474 was renamed to  "The init container now has the same resource requests as the main container of a correspondent Pod (Thanks to user yann.leenhardt for contribution)" after the consultation with @qJkee